packages/rocketchat-lib/client/MessageAction.coffee

```diff
@@ -79,6 +79,33 @@ Meteor.startup ->
 			RocketChat.MessageAction.hideDropDown()
 
 	RocketChat.MessageAction.addButton
+		id: 'reply-message'
+		icon: 'icon-reply'
+		i18nLabel: 'Reply'
+		context: [
+			'message'
+			'message-mobile'
+		]
+		action: (event, instance) ->
+			message = @_arguments[1]
+			input = instance.find('.input-message')
+			url = RocketChat.MessageAction.getPermaLink(message._id)
+			console.log message
+			text = '[ ](' + url + ') @' + message.u.username + ': '
+			if input.value
+				input.value += if input.value.endsWith(' ') then '' else ' '
+			input.value += text
+			input.focus()
+			$(input).keyup()
```

Comment:
Hmm.. I have copied from the [quote action](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-lib/client/MessageAction.coffee#L215). I will try checking.